documentation/src/main/asciidoc/public/reference/_search-dsl-projection.adoc

```diff
@@ -1502,6 +1503,29 @@ include::{sourcedirJava17}/org/hibernate/search/documentation/search/projection/
 populated with the requested highlights and field.
 ====
 
+.Returning highlights for matched documents within a projection constructor. Single highlighted fragment
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedirJava17}/org/hibernate/search/documentation/search/projection/MyBookHighlightedTitleProjection.java[tags=include]
+----
+<1> Annotate the record type with <<mapping-projection-basics,`@ProjectionConstructor`>>.
+<2> Annotate the parameter that should receive the highlights value with `@HighlightProjection`.
+<3> The type of the constructor parameter, in this case, must be assignable from `String`,
+since we are planning to return a single highlighted fragment.
+Note, that only a highlighter that calls the <<search-dsl-highlighting-highlighter-options-fragment-size,`.numberOfFragments(1)`>>
+can be used for single-valued highlight projections.
+
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedirJava17}/org/hibernate/search/documentation/search/projection/ProjectionDslJava17IT.java[tags=projection-constructor-highlight-single]
+----
+<1> Pass the custom projection type to `.select(...)`.
+<2> Configure the highlighter to return a single highlighted fragment.
+<3> Each hit will be an instance of the custom projection type,
+populated with the requested highlight and field.
+====
+
```

Comment:
I didn't want to add a single highlight to the previous example as that would mean that we'd need to make a query more complex (matching on multiple fields)
But then I can't say that I really like this approach with two similar examples 🙈 ...